src/main/java/serverutils/handlers/ServerUtilitiesServerEventHandler.java

```diff
@@ -182,7 +184,13 @@ public void onServerTick(TickEvent.ServerTickEvent event) {
                     boolean isAFK = data.afkTime >= ServerUtilitiesConfig.afk.getNotificationTimer();
 
                     if (isAFK) {
-                        player.addStat(ServerUtilitiesStats.AFK_TIME, 1);
+                        if (!prevIsAfk) {
+                            player.addStat(
+                                    ServerUtilitiesStats.AFK_TIME,
+                                    floor(ServerUtilitiesConfig.afk.getNotificationTimer() / 50));// Millisec to ticks
```

Comment:
One issue I can see here that this assumes the server runs at perfect 20 TPS, if it runs any slower this will overcount AFK timing by quite a bit

Reply:
Shouldn't time be counted by irl time and not in-game time? How is it done right now?

Reply:
The stat counter increments by 1 every tick someone is AFK, same as the actual playtime statistic which is also tps-bound - but the afk notification timer is based on real time, not tick time. I don't think there's an easy way around this, without building a more complicated tracker that also keeps track of the tick number that someone went afk